flexibleengine/acceptance/resource_flexibleengine_rds_account_test.go

```diff
@@ -0,0 +1,226 @@
+package acceptance
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	model "github.com/huaweicloud/huaweicloud-sdk-go-v3/services/rds/v3/model"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccRdsAccount_basic(t *testing.T) {
+	rName := acceptance.RandomAccResourceName()
+	resourceName := "flexibleengine_rds_account.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: TestAccProviderFactories,
+		CheckDestroy:      testAccCheckRdsAccountDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testRdsAccount_basic(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckRdsAccountExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "name", rName),
+					resource.TestCheckResourceAttr(resourceName, "password", "Test@12345678"),
+				),
+			},
+			{
+				Config: testRdsAccount_update(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckRdsAccountExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "name", rName),
+					resource.TestCheckResourceAttr(resourceName, "password", "Test@123456789"),
+				),
+			},
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"password"},
+			},
+		},
+	})
+}
+
+func testAccCheckRdsAccountDestroy(s *terraform.State) error {
+	c := acceptance.TestAccProvider.Meta().(*config.Config)
```

Comment:
please use testAccProvider to get the config
```
c := testAccProvider.Meta().(*config.Config)
```

Reply:
fixed